demo/java-demo-client/src/main/java/org/cloudfoundry/abacus/demo/model/Usage.java

```diff
@@ -0,0 +1,128 @@
+package org.cloudfoundry.abacus.demo.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Arrays;
+import java.util.StringJoiner;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Usage {
+
+  private long start;
```

Comment:
Add `@JsonProperty` annotation to `start` and `end` for consistency and refactor-rename protection.

Reply:
Done